packages/react-components/src/components/SendBox.tsx

```diff
@@ -217,12 +242,36 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   /* @conditional-compile-remove(file-sharing) */
-  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+  const uploadInProgressErrorBar = useMemo(() => {
+    const renderError = !!(
+      props.activeFileUploads?.length &&
+      !props.activeFileUploads
+        .filter((fileUpload) => !fileUpload.errorMessage)
+        .every((fileUpload) => fileUpload.isUploaded)
+    );
+    return renderError && <SendBoxErrorBar message={strings.fileUploadsPendingError} />;
+  }, [props.activeFileUploads, strings.fileUploadsPendingError]);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const fileUploadErrorBar = useMemo(() => {
     const fileUploads: ActiveFileUpload[] = props.activeFileUploads || [];
     const latestError = fileUploads.filter((fileUpload) => fileUpload.errorMessage).pop();
-    return <SendBoxErrorBar message={latestError?.errorMessage} timeout={10 * 1000} />;
+    return latestError && <SendBoxErrorBar message={latestError.errorMessage} timeoutDelay={10 * 1000} />;
   }, [props.activeFileUploads]);
 
+  /* @conditional-compile-remove(file-sharing) */
+  const renderSendBoxError = () => {
+    if (showFileUploadsPendingError) {
+      fileUploadsPendingErrorTimeoutRef.current && clearTimeout(fileUploadsPendingErrorTimeoutRef.current);
+      fileUploadsPendingErrorTimeoutRef.current = setTimeout(() => {
```

Comment:
What is the difference between this timeout and the timeoutDelay in SendBoxErrorBar?

Reply:
This timeout is used to hide the upload in progress error after 10 seconds.
We can't use sendboxerror timeout for this because not only do we need to hide the error after 10 seconds, we also need to ensure that it is not rendered again until someone tries to send the message.
(there is a state variable for that `showFileUploadsPendingError` which is set to false after 10 seconds using this timeout)